metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -235,7 +235,7 @@ function getDisabledInstrumentationsFromEnv() {
 export function getResourceDetectorsFromEnv(): Array<Detector | DetectorSync> {
   const resourceDetectors = new Map<
     string,
-    Detector | DetectorSync | Detector[] | DetectorSync[]
+    Detector | DetectorSync | Detector[] | DetectorSync[] | Array<Detector | DetectorSync>
```

Comment:
note for reviewer: this is necessary to allow this single refactor instead of doing all AWS detector at once.